examples/object_detection/main.py

```diff
@@ -290,7 +290,10 @@ def train(net, compression_ctrl, train_data_loader, test_data_loader, criterion,
     best_compression_level = CompressionLevel.NONE
     test_freq_in_epochs = max(config.test_interval // epoch_size, 1)
 
+    last_epoch = -1 if config.start_iter == 0 else config.start_iter // epoch_size
+    compression_ctrl.scheduler.epoch_step(last_epoch)
```

Comment:
Why do we set `last_epoch` to -1 if we call then `compression_ctrl.scheduler.epoch_step(last_epoch)` which doesn't  increment it.

Reply:
Fixed.